packages/yuudachi/src/functions/anti-raid/filters.ts

```diff
@@ -67,10 +67,10 @@ export function avatarFilter(member: GuildMember, avatar?: string | null) {
 	}
 
 	if (avatar.toLowerCase() === 'none') {
-		return !Boolean(member.user.avatar);
+		return !member.avatar && !member.user.avatar;
 	}
 
-	return member.user.avatar === avatar;
+	return member.user.avatar === avatar || member.avatar === avatar;
```

Comment:
Can you swap the condition here, just to be consistent with the above. Or swap the above.